### ISSUE TITLE
Pillar Box issue on macOS 13.2.1

### ISSUE THREAD

**Issue by olivergiangvu** [Aidoku/Aidoku#204]
I have problem in the new pillar box feature for webtoon in the macOS (13.2.1) - it work fine with my ipad. Pic res not dynamic change like before  and it seem to be stretched out not keep aspect ratio anymore  turn on pillar box fix it but the landscape pillar box was too huge.

![CleanShot_2023-03-29_at_08 57 172x](https://user-images.githubusercontent.com/106240777/228407661-9db8db86-8025-4899-bf99-c072c81e10c6.jpg)
![CleanShot_2023-03-29_at_08 57 452x](https://user-images.githubusercontent.com/106240777/228407679-beee6f39-08c2-4c81-a0f3-d104d737c3b3.jpg)
![CleanShot_2023-03-29_at_08 58 152x](https://user-images.githubusercontent.com/106240777/228407690-f7c90fbe-18aa-424d-bbc0-683d6b371659.jpg)
![CleanShot_2023-03-29_at_08 58 342x](https://user-images.githubusercontent.com/106240777/228407906-fba304d1-7b45-4330-b751-7a38d357ee1a.jpg)

**Comment by olivergiangvu** [comment]
> Interesting. I don't optimize Aidoku for macOS whatsoever, but I'll try and take a look at this.

Thanks! I have reverted to last build 0.6.1(7) - works shamelessly on macOS 13.3 RC. The pic dynamic res scaling work well.

**Comment by TheFermi0n** [comment]
### Adding to the issue

In the latest version `Version 0.6.4 (3)` the scaling is fixed. I can resize the app window and the Webtoon pages resize itself properly.

### Still present

When you resize it adds a lot of empty space between pages as seen in the attached screen recording.

https://github.com/Aidoku/Aidoku/assets/63984668/7f976aa0-f559-4b85-b272-c03119d32778

P.S. The pillar box size does not matter. It is same for any size.

**Comment by Skittyblock** [comment]
Looks like
- when not using pillarbox, the scaling breaks when you resize the window and squishes the pages
- when using pillarbox, it'll resize correctly but with spaces between the pages to a point, and then it'll start stretching if it gets too large.

Unfortunately looks like an issue with how AsyncDisplayKit does cell sizing on macOS for some strange reason, I'm not sure why it doesn't properly use the ratio, but I would probably have to spend a lot of time either looking for a workaround to properly refresh the cell sizes or reimplement the correct behavior myself. Not a priority for me at all though since the eventual plan is to have a separate, native macOS app.

**Comment by TheFermi0n** [comment]
> Not a priority for me at all though since the eventual plan is to have a separate, native macOS app.

Yeah, no major bugs in it right now. It is usable. Looking forward to the native macOS client. That will be much better

**Comment by Skittyblock** [comment]
fixed by ec4018c0f3dd62c2ce99c7ef0f36207c97f873b1
